pages/apis/rest_api/packages/registries.md

```diff
@@ -2,6 +2,56 @@
 
 The registries API lets you create and manage registries in your organization.
 
+## Create a registry
+
+```bash
+curl -H "Authorization: Bearer $TOKEN" \
+  -X POST "https://api.buildkite.com/v2/packages/organizations/#{org.slug}/registries" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "name": "my registry",
+    "ecosystem": "ruby"
+    }' 
+```
+
+```json
+{
+  "id": "0191df84-85e4-77aa-83ba-6579084728eb",
+  "graphql_id": "UmVnaXN0cnktLS0wMTkxZGY4NC04NWU0LTc3YWEtODNiYS02NTc5MDg0NzI4ZWI=",
+  "slug": "my-registry",
+  "url": "https://api.buildkite.com/v2/packages/organizations/my-org/registries/my-registry",
+  "web_url": "https://buildkite.com/organizations/my-org/registries/my-registry",
+  "name": "my registry",
+  "ecosystem": "ruby",
+  "description": null,
+  "emoji": null,
+  "color": null,
+  "public": false,
+  "oidc_policy": null
+}
+```
+
+Required [request body properties](/docs/api#request-body-properties):
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>name</code></th><td>Name of the new registry.<br><em>Example:</em> <code>"my registry"</code>.</td></tr>
+  <tr><th><code>ecosystem</code></th><td>Registry ecosystem based on the <a href="/docs/packages#get-started">package ecosystem</a> for the new registry.<br><em>Example:</em> <code>"ruby"</code>.</td></tr>
+</tbody>
+</table>
+
+Optional [request body properties](/docs/api#request-body-properties):
+
+<table class="responsive-table">
+  <tbody>
+    <tr><th><code>description</code></th><td>Description of the registry.<br><em>Default value:</em> <code>null</code>.</td></tr>
+  </tbody>
+</table>
+
+Required scope: `create_registries`
```

Comment:
`create_registries` doesn't exist. I think this should be [write_registries](https://github.com/buildkite/buildkite/blob/6577aa6bf6e59191bf00a0a6495205a392406a36/app/models/api/access_token/scopes.rb#L48).

Reply:
Thank you for catching this!